### PR TITLE
27 Buyer should view recommended products

### PR DESCRIPTION
**Pull request by Evaristekanova**
### what does this PR do?

- recommend product to the buyers

### description of task to completed

- have this endpoint working GET "/api/v1/buyers/view/recommendations"
- how should this be manually tested
- clone this repo
- switch in ft-buyer-recommended-products-27
- open terminal
- run "npm i" to install all dependencies
- run "npm run dev" to start server
- open postman and use the above route
- open documentation with this link "localhost:{{port}}/docs"

### trello story number

- number 18

### 
Screenshot 2023-03-24 112130
screenshot

### CODE REVIEW

**Comment by mansurissa** [comment]
Improve your PR description, I shared markdown resource that may help you.

File: src/controllers/products/buyerViewSingleProductController.js [hunk 1]
```diff
@@ -23,6 +25,39 @@ export const viewSingleProduct = asyncWrapper(async (req, res) => {
...
+  const { id } = req.user;
+  const wishlist = await db.wishlist.findOne({ where: { userId: id }, attributes: ['productId'] });
+  if (!wishlist) {
+    return res.status(404).json({
+      status: req.t('fail'),
+      message: req.t('no_wishlist'),
+    });
+  }
+  const { productId } = wishlist.dataValues;
+  const reco = await productId.map(async (id) => {
```
Comment (mansurissa) [inline src/controllers/products/buyerViewSingleProductController.js]: Work on your naming conventions (Use names that are easily understandable and use camelCase) like 
1. `reco`, I think you wanted to refer to recommendation
2. whishlist : `wishList` and so on...

**Review by BenoitIT — APPROVED** [review]

File: src/controllers/products/buyerViewSingleProductController.js (hunk 1 above)
Comment (mansurissa) [reply src/controllers/products/buyerViewSingleProductController.js]: We can arrange a call tomorrow morning to share ideas on this algorithm of product recommendations

**Review by mansurissa — CHANGES_REQUESTED** [review]

File: src/controllers/products/buyerViewSingleProductController.js (hunk 1 above)
Comment (Evaristekanova) [reply src/controllers/products/buyerViewSingleProductController.js]: sure we can do that but I have changed variables and now I am using readable variable

**Review by FabCode67 — APPROVED** [review]
good job @Evaristekanova

**Pull request merged**